src/box.tsx

```diff
@@ -14,6 +15,36 @@ const Box: BoxComponent = ({ is = 'div', innerRef, children, ...props }) => {
     parsedProps.ref = innerRef
   }
 
+  /**
+   * If the user has enabled safe hrefs we want to make sure that the url passed
+   * is safe and that the other attributes that make the link safe are added to the
+   * element
+   */
+  const safeHrefEnabled = !allowUnsafeHref && getUseSafeHref() && is === 'a' && parsedProps.href
```

Comment:
`!allowUnsafeHref` is a breaking change because by default it will be `undefined` which is falsy. I think we either want to set the default value to `true` for now OR change the logic here to explicitly check that it's a boolean value.

Reply:
That should still work as intended even if the value is undefined(`falsy`) unless I'm not thinking about it right

Reply:
hm, I guess it will be saved by the fact that `getUseSafeHref` is `false` by default. But `!allowUnsafeHref` will look true when the user _has not defined a prop_.

Thinking more about this condition it doesn't meet these requirements:
- if a prop is specified `true|false` it should take precedence over the global configuration
- else if the global configuration is set, use that
- else default to "off" so we are backwards compatible and the change is fully opt-in.

Reply:
`(typeof allowUnsafeHref === 'boolean' ? !allowUnsafeHref : getUseSafeHref()) && ...`

Reply:
Ahh I see what you mean, that's smart too as it would allow the dev to do the reverse flow of what I was thinking, which would be have safe links disabled globally and overwrite on a specific use case

Reply:
Done! :)